src/protagonist/Engine/Ingest/Image/Appetiser/Models/AppetiserResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Engine.Ingest.Image.Appetiser;
+
+public class AppetiserResponse
+{
+    public ResponseType ResponseType { get; init; }  
```

Comment:
This field doesn't appear to be used - can it be removed?

Reply:
I can remove this field, but this was an attempt to avoid a marker interface - essentially I was choosing which was less bad